app/controllers/data_access_controller.rb

```diff
@@ -182,9 +182,9 @@ def options
         response.headers[key] = value if key.start_with?('cmr-')
       end
 
-      respond_with(result, status: catalog_response.status)
+      respond_with(result, status: catalog_response.status, location: nil)
```

Comment:
It will otherwise throw an location related error because "result" is not an ActiveRecord. Can't remember the exact error though.